auto_round/mllm/autoround_mllm.py

```diff
@@ -160,6 +160,9 @@ def __init__(
                 self.template, model=model, tokenizer=tokenizer, processor=processor, image_processor=image_processor)
             dataset = self.template.default_dataset if dataset is None else dataset
         
+        if model.config.model_type == "deepseek_vl_v2":
```

Comment:
the setting here is a little tricky.  Could the quantizing-non-text-module still be supported?